src/conversion_helper.js

```diff
@@ -699,7 +710,7 @@ function findPotentialConversions(comment) {
     if (duplicateCache[unit] === undefined) {
       duplicateCache[unit] = [number];
       return true;
-    } else if (duplicateCache[unit].indexOf(number) === -1) {
+    } else if (!ContainsArray(duplicateCache[unit], number)) {
```

Comment:
Ah, here is where the confusion came from. The variable is still named "number" but it's actually an array now - but I understand you're still in the middle of working on it so not everything is refactored yet :)